### DISCUSSION TITLE
Mirror option

### DISCUSSION THREAD

**Discussion by Trioplane** [Yqnn/svg-path-editor#38] [Ideas]
Ive been wanting a mirror option for so long so its gonna be easier to mirror one side to the other side

**Comment by AnweshGangula** [comment]
You should be able to achieve this by scaling the existing path in negative direction like
- `ScaleX: -1` to mirror horizontally
- `ScaleY: -1` to mirror vertically

Then you can copy the mirrored path and add it below the original one
